Cargo.toml

```diff
@@ -3,6 +3,7 @@
 name = "pcap"
 version = "0.7.0"
 authors = ["Sean Bowe <ewillbefull@gmail.com>"]
+edition = "2018"
```

Comment:
You might want to mention it in Changelog entry.

Reply:
Ah, I had this on my branch, but strictly speaking we don't need to update the version. I don't know what potential this has to be breaking though.